static/js/flow/ecommerceTypes.js

```diff
@@ -113,10 +126,10 @@ export type Product = {
   id: number,
   title: string,
   product_type: string,
-  created_on: Date,
-  updated_on: Date,
-  object_id: number,
-  content_type: number,
+}
+
+export type ProductDetail = Product & {
+  latest_version: BasketItem,
```

Comment:
Should this be `ProductVersion` instead of `BasketItem`?

Reply:
Ah yes, fixed